src/kibana/components/vislib/styles/_tilemap.less

```diff
@@ -116,4 +120,8 @@ img.leaflet-tile {
 
 img.leaflet-tile.filters-off {
   .filter(none);
+}
+
+.leaflet-clickable {
```

Comment:
This removes mouse events on the svg elements to allow the revised tooltip methods to work.